### PR TITLE
111 in the committee dashboard create a page where committee members can send out custom messages

### PR DESCRIPTION
**Pull request by fredrir**
- ApplicantCard skeleton loader
- Fixed Application form error when submiting
- Replaced ApplicationOverview with ApplicantCard
- Added a page in the committee dashboard where committees can add custom messages
- A message value is now added to committee times
- Refactored the dashboard UI to be less confusing
- Refactored the API routes for committees, in order to only fetch the necessary data 
- Fixed a bug with the application form related to how optionalCommittees where initialized

### CODE REVIEW

File: components/committee/CommitteeInterviewTimes.tsx [hunk 1]
```diff
@@ -15,51 +15,75 @@ interface Interview {
   end: string;
 }
 
+interface Props {
+  periode: periodType | null;
```
Comment (julian-ao) [inline components/committee/CommitteeInterviewTimes.tsx:19]: Any reason this says "periode"?

File: components/applicantoverview/ApplicantCard.tsx [hunk 2]
```diff
@@ -15,6 +15,11 @@ const ApplicantCard = ({ applicant, includePreferences }: Props) => {
   };
 
   const preferences = applicant?.preferences || {};
+  const filteredPreferences = Object.entries(preferences).filter(
+    ([key, value]) => value
+  );
+
+  console.log(applicant);
```
Comment (julian-ao) [inline components/applicantoverview/ApplicantCard.tsx:22]: regner med at den ikke skulle være med?

**Comment by julian-ao** [comment]
No way jeg reviewer 1500 linjer med kode, så approver bare nå, det ser sikkert nice ut 😂 Fint om du deler opp PRs til mindre neste gang. En PR endrer kun en ting liksom,

**Review by julian-ao — APPROVED** [review]

**Comment by fredrir** [comment]
> No way jeg reviewer 1500 linjer endret, så approver bare nå, det ser sikkert nice ut 😂 Fint om du deler opp PRs til mindre neste gang. En PR endrer kun en ting liksom.

Skal deles opp til neste gang🚀🚀

**Pull request merged**